### ISSUE TITLE
Test-Vault fails.

### ISSUE THREAD

**Issue by jdhitsolutions** [TylerLeonhardt/SecretManagement.LastPass#29]
Not sure why this fails.

![test-vault](https://user-images.githubusercontent.com/12089920/114785529-d44eb200-9d4a-11eb-8d06-8a6ca7b21102.png)

The vault and extension work fine otherwise.

**Comment by TylerLeonhardt** [comment]
Here's the logic for it:

https://github.com/TylerLeonhardt/SecretManagement.LastPass/blob/main/SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1#L398-L409

what does `lpass status` return for you.

**Comment by jdhitsolutions** [comment]
I have to run `wsl lpass status` in Windows and I get a "not logged in" message. Even though I can run `Get-Secret` and `Get-SecretInfo` from my lastpass vault.

**Comment by jdhitsolutions** [comment]
I went into WSL and logged in to Lastpass. Then back in Windows `Test-SecretVault` works.  Your test code is odd because even though the test technically fails, I can still extract secrets from the vault. Or it would be helpful to have more verbose detail about what `Test-SecretVault` is actually doing.

**Comment by TylerLeonhardt** [comment]
Yeah I'm not 100% sure what `Test-SecretVault` should really be doing. I'm tempted to always return true but I want it to provide some value.

**Comment by jdhitsolutions** [comment]
From what I can infer, the test can be anything you want. I would assume it is to ensure "integrity" which in this case I'd have to think means being able to read from the vault, or somehow verifying that the vault isn't corrupt. `lpass status` doesn't seem to offer much value.  Maybe verifying that `lpass ls` doesn't throw an error?

**Comment by TylerLeonhardt** [comment]
Yeah that's probably the best bet. Would you like to contribute that? I'm going to need to find a new maintainer eventually for this repo as I don't use LastPass anymore :(

**Comment by jdhitsolutions** [comment]
I can try to give this a go. I'll have to get set up to submit PRs to the repo.
